src/integration-test/tests/citizen/claimantResponse/part_admission_test.ts

```diff
@@ -63,6 +63,8 @@ if (process.env.FEATURE_ADMISSIONS === 'true') {
     confirmationPage.clickGoToYourAccount()
     I.see(testData.claimRef)
     I.see('You’ve signed a settlement agreement.')
+    I.click(testData.claimRef)
+    I.see('you need to tell us')
```

Comment:
Is that sufficient test 'you need to tell us ??' - do we need more info here that require testing?
And other places as well.

Reply:
I can add a call to an I.click() test of the hyperlink, for each of these scenarios

Reply:
After discussing with Rob, it is not necessary to test the anchor. We are just testing to ensure the dashboard status appears. There is another separate test that checks the hyperlink works as expected.